worker/src/main/scala/org/apache/celeborn/service/deploy/worker/PushDataHandler.scala

```diff
@@ -76,6 +77,7 @@ class PushDataHandler extends BaseMessageHandler with Logging {
     storageManager = worker.storageManager
     shutdown = worker.shutdown
     conf = worker.conf
+    workerPartitionSplitEnabled = conf.workerPushHeartbeatEnabled
```

Comment:
typo? should be workerPartitionSplitEnabled

Reply:
yes. it's my fault. it has been corrected.